### PR TITLE
cmake: Cleanup usage of option

### PR DESCRIPTION
**Pull request by juan-lunarg**
By default option is OFF

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -59,16 +59,15 @@ if ("${CMAKE_BUILD_TYPE}" STREQUAL "")
     set(CMAKE_BUILD_TYPE "Debug")
 endif()
 
-option(SKIP_GLSLANG_INSTALL "Skip installation" ${SKIP_GLSLANG_INSTALL})
```
Comment (juan-lunarg) [inline CMakeLists.txt:62]: This is actively confusing and hard to read.

**Review by ncesario-lunarg — COMMENTED** [review]
Seems fine to me, but at the same time is there any downside to making the initial value explicit?

**Comment by juan-lunarg** [comment]
> Seems fine to me, but at the same time is there any downside to making the initial value explicit?

It's less idiomatic.

File: CMakeLists.txt (hunk 1 above)
Comment (juan-lunarg) [inline CMakeLists.txt:62]: This is the main reason I made this PR. It's difficult to tell by reading the code that SKIP_GLSLANG_INSTALL is OFF by default.

Comment (ncesario-lunarg) [reply CMakeLists.txt:62]: Yeah, I agree this was confusing and should change. I'm not sure I agree that the proposed change is clearer and/or more idiomatic than `option(SKIP_GLSLANG_INSTALL "Skip installation" OFF)`, but I'll leave that up to @arcady-lunarg :)

Comment (juan-lunarg) [reply CMakeLists.txt:62]: I don't mind either way. I just use this approach in every other repo I work on.

Comment (arcady-lunarg) [reply CMakeLists.txt:62]: Judging by the cmake book, not having "OFF" is more idiomatic.

**Pull request merged**